File: shoring/flywheel/src/test/java/com/panopset/tests/flywheel/TemplateTest.java
package com.panopset.tests.flywheel;

import java.io.IOException;

public final class TemplateTest {

  /**
   * Test template command. See <b>templateTest01.txt</b>.
   */
  public void testTemplateCommand() throws IOException {
    new SimpleTest().comparisonTest("templateTest01.txt", "templateTest.txt",
        "templateTestExpected.txt");
  }

}

File: code/shoring/flywheel/src/test/kotlin/com/panopset/tests/flywheel/TemplateTest.kt
package com.panopset.tests.flywheel

import org.junit.jupiter.api.Test

class TemplateTest {
    /**
     * Test template command. See **templateTest01.txt**.
     */
    @Test
    fun testTemplateCommand() {
        SimpleTest().comparisonTest(
            "templateTest01.txt", "templateTest.txt",
            "templateTestExpected.txt"
        )
    }
}
